SPECS/perl/perl.spec

```diff
@@ -6847,6 +6838,10 @@ popd
 
 # Old changelog entries are preserved in CVS.
 %changelog
+* Thu Feb 15 2024 Pawel Winogrodzki <pawelwi@microsoft.com> - 4:5.38.2-504
+- Updated patch application macros.
+- Applying missed patches 14 and 202.
```

Comment:
Do we know why these were excluded?

Reply:
Yes, I spoke with @nicogbg. Turns out we accidentally missed them during an upgrade but there was no reason to skip them.